pkg/session_recording/recordings.jsx

```diff
@@ -398,6 +491,35 @@
             this.journalctl = null;
         }
 
+        /*
+         * Restarts journalctl.
+         * Assumes journalctl is running.
+         */
+        journalctlRestart() {
+            if(this.journalctlIsRunning()) {
+                this.journalctl.stop();
+                this.recordingMap = {};
+                this.setState({recordingList: []});
+            }
+            this.journalctlStart();
+        }
+
+        /*
+         * Handles Datepicker Since.
+         */
+        handleDateSinceChange(date) {
+            this.setState({dateSince: date});
+            this.journalctlRestart();
```

Comment:
Event on date change won't fire before date is actually changed. I think we can keep it as is for now, but I will think about better way for future.